content/resources/user/gen3-client.md

```diff
@@ -52,60 +53,105 @@ Now you should be able to run the tool on the command-line from any directory by
 Before using the gen3-client to upload or download data, the gen3-client needs to be configured with API credentials downloaded from the user's data commons Profile:
```

Comment:
Is it clear that user would need to switch from command line to Windmill for this step?